js/wasm/src/webworker/index.ts

```diff
@@ -80,15 +80,6 @@ async function initializeEnvironment(
 	updateProgress("Loading Gradio wheels");
 	await pyodide.loadPackage(["ssl", "setuptools"]);
 	await micropip.add_mock_package("ffmpy", "0.3.0");
-	await micropip.install.callKwargs(
-		[
-			"typing-extensions>=4.8.0", // Typing extensions needs to be installed first otherwise the versions from the pyodide lockfile is used which is incompatible with the latest fastapi.
```

Comment:
`typing-extensions` bundled in Pyodide has been updated (https://github.com/pyodide/pyodide/pull/4428, https://github.com/pyodide/pyodide/pull/4712) so this line is no longer needed.